### ISSUE TITLE
Upgrading from Kubic Ubuntu packages overwrites containers.conf

### ISSUE THREAD

**Issue by c-goes** [containers/podman#9530]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

I'm using Podman in WSL2 and have to change the containers.conf not to use systemd/journald after upgrades to make it work.

**Steps to reproduce the issue:**

1. Upgrade to newer  Podman version after containers.conf is changed

**Describe the results you received:**

containers.conf is overwritten by new package version

**Describe the results you expected:**

Config should stay the same or ask the user to merge the new config.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

This happend with upgrade to 3.0.1-2

**Output of `podman info --debug`:**

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide?**

Yes

**Additional environment details (AWS, VirtualBox, physical, etc.):**

WSL2

**Comment by mheon** [comment]
Which containers.conf - in /usr/share or /etc? It is intended that the one
in usr/share be overwritten. User changes should be made in /etc/containers.

**Comment by c-goes** [comment]
> Which containers.conf - in /usr/share or /etc? It is intended that the one in usr/share be overwritten. User changes should be made in /etc/containers.

I just noticed that on my system /etc/containers/containers.conf is a symlink to /usr/share/containers/containers.conf (probably created from an older version of the package). I didn't expect this.
This symlinking isn't done anymore since  October or November. Thus, for new installations this should work now as expected. I ran `apt purge` on all the containers packages, removed the symlinks and reinstalled and it's working now.
